modules/sandboxed-containers-deleting-kataconfig-cli.adoc

```diff
@@ -6,14 +6,27 @@
 [id="sandboxed-containers-deleting-kataconfig-cli_{context}"]
 = Deleting the KataConfig custom resource using the CLI
 
-Remove and uninstall the `kata` runtime and all its related resources, such as CRI-O config and `RuntimeClass`, from your cluster. The deletion typically takes between ten to forty minutes, depending on the size of the deployment.
+Remove and uninstall the `kata` runtime and all its related resources, such as CRI-O config and `RuntimeClass`, from your cluster.
 
 .Prerequisites
 
 * You have {product-title} {product-version} installed on your cluster.
 * You have installed the OpenShift CLI (`oc`).
 * You have access to the cluster as a user with the `cluster-admin` role.
 
+[IMPORTANT]
+====
+Deleting the `KataConfig` CR automatically reboots the worker nodes. The reboot can take anywhere from ten minutes to over an hour. Factors that impede reboot time are as follows:
+
+* A larger {product-title} deployment with a greater number of worker nodes.
+* Activation of BIOS and Diagnostics.
+* Deployment on a hard drive rather than an SSD.
+* Installation on physical nodes such as bare metal, rather than on virtual nodes.
+* Slow CPU and network.
+
+During the reboot, the CLI window freezes. Wait for the process to complete before continuing to the next procedure.
```

Comment:
During deletion, the CLI freezes until all the nodes reboot. Wait for the process to complete before continuing to the next procedure.